src/infrastructure/http/deps.py

```diff
@@ -0,0 +1,17 @@
+from fastapi import Request
+
+from apps.shared.errors import ForbiddenError
+from infrastructure.http.domain import MindLoggerHeaders
+
+
+async def mindlogger_headers(request: Request) -> MindLoggerHeaders:
```

Comment:
Could you implement this function as `get_current_user` to add it using dependency injection

Reply:
Sure